src/js/image.js

```diff
@@ -190,7 +190,11 @@ $.magnificPopup.registerModule('image', {
 				var img = document.createElement('img');
 				img.className = 'mfp-img';
 				if(item.el && item.el.find('img').length) {
-					img.alt = item.el.find('img').attr('alt');
```

Comment:
Rather than an `if` statement with mostly duplicate lines, simpler to tack on `|| ''`:

    img.alt = item.el.find('img').attr('alt') || '';

Reply:
Sure! I guess I'm always leery of depending on truthiness in loose-typed languages. Maybe because I'm too accustomed to PHP's terrible string finagling.